flocker/node/agents/test/test_blockdevice.py

```diff
@@ -148,19 +148,35 @@
     blockdevice_id=_create_blockdevice_id_for_test(dataset.dataset_id),
 ))
 
+DESIRED_DATASET_ATTRIBUTE_STRATEGIES = {
+    'dataset_id': uuids(),
+    'maximum_size': integers(min_value=0).map(
+        lambda n:
+        LOOPBACK_MINIMUM_ALLOCATABLE_SIZE
+        + n*LOOPBACK_ALLOCATION_UNIT),
+    'metadata': dictionaries(keys=text(), values=text()),
```

Comment:
Unfortunately it's still 60 seconds of runtime, but I'm sure it'll be nicer in logs.